frontend/ui/quickstart.lua

```diff
@@ -10,61 +10,185 @@ local lfs = require("libs/libkoreader-lfs")
+
+<div class="img-block break-before-avoid">![Default setting 1](../resources/quickstart/defaultsetting1.png)</div>
+<div class="img-block break-before-avoid">![Default setting 2](../resources/quickstart/defaultsetting2.png)</div>
+
+- You can see explanations for all items on the **BOTTOM MENU** by tapping and holding the option’s name. This is also available for most of the **TOP MENU** menu items.
+
+- You can **CLOSE** full screen dialogs (History, Table of Contents, Bookmarks, Reading Statistics etc.) by swiping down
+
+- **SCREENSHOTS** can be taken by touching opposing corners of the screen diagonally at the same time or by making a long diagonal swipe
+
+<div class="img-block break-after-avoid">![Number picker](../resources/quickstart/numberpicker.png)</div>
+
+- In dialogs containing adjustment arrow buttons like the one above, you can tap and hold on arrow buttons to increase / decrease the value in bigger increments
+
+- You can **CLOSE** this type of dialog (non-full screen) by tapping outside of the window. You can **MOVE** this type of dialog by holding the window title and dragging
+
+- You can make this type of dialog **SEMI-TRANSPARENT** (to see the text under it while adjusting a value) by tapping and holding the window title
+
+- Tapping and holding a word brings up a dialog where you can also search the selection in the document for more occurrences or look it up on Wikipedia
+
+- You can highlight sections by tapping and holding a word and dragging your finger
+
+- You can move through your document via **SKIM DOCUMENT** dialog:
+
+> **TOP MENU ➔ ![Navigation](../resources/icons/mdlight/appbar.navigation.svg) ➔ Skim document**
+
+## ACCESSING FILES <a id="afiles"></a>
+
+The following methods are available for accessing your books and articles:
+
+- File Browser
+- Favorites
+- History
+
+You can assign gestures for each of these dialogs.
+
+You can also set KOReader to open with any of these dialogs on startup via:
+
+> **TOP MENU (in File Browser) ➔ ![Filebrowser](../resources/icons/mdlight/appbar.filebrowser.svg) ➔ Start with**
+
+## TRANSFERRING FILES <a id="tfiles"></a>
+
+In addition to transferring files the same way you would with the built-in reader application, other options are available depending on your device:
+
+1. USB Mass Storage mode within KOReader
+2. Cloud storage (Dropbox/FTP/Webdav)
+3. SSH/SFTP access
+4. Calibre transfer
+5. News downloader
+6. Wallabag
+
+## FRONTLIGHT/BACKLIGHT <a id="flight"></a>
+
+You can control your screen light via this menu. If you have warm lighting (normal white LEDs+orange ones) you can control them separately from this dialog:
+
+> **TOP MENU ➔ ![Settings](../resources/icons/mdlight/appbar.settings.svg) ➔ Frontlight**
```

Comment:
We have access to Device:hasNaturalLight(). Whether to show this depends a bit on if we want to advertise certain Kobo devices, I suppose.